superset/assets/spec/javascripts/components/ColumnTypeLabel_spec.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+import { shallow } from 'enzyme';
+
+import ColumnTypeLabel from '../../../javascripts/components/ColumnTypeLabel';
+
+describe('ColumnOption', () => {
+  const defaultProps = {
+    type: 'string',
+  };
+
+  let wrapper;
+  let props;
+  const factory = o => <ColumnTypeLabel {...o} />;
```

Comment:
I know there are a lot of places that do this already, but single character variable names don't facilitate readability.